test/mocha/68-election-get-ancestors.js

```diff
@@ -144,18 +141,19 @@ describe('Election API _getAncestors', () => {
     async.auto({
       event1: callback => helpers.addEventAndMerge(
         {consensusApi, eventTemplate, ledgerNode, opTemplate}, callback),
-      test: ['event1', (results, callback) => {
+      test: ['event1', async results => {
         const hashes = {
           mergeEventHashes: [results.event1.mergeHash],
           parentHashes: results.event1.merge.event.parentHash
         };
-        getAncestors({hashes, ledgerNode}, (err, result) => {
-          assertNoError(err);
+        try {
+          const result = await getAncestors({hashes, ledgerNode});
           should.exist(result);
           result.should.be.an('array');
           result.should.have.length(2);
-          callback();
-        });
+        } catch(e) {
+          assertNoError(e);
```

Comment:
async func, can remove try/catch and let error propagate out.

Reply:
I'm not touching the tests (as much as possible) in this pass.